GOV_UK_API_ISSUES

```diff
@@ -0,0 +1,57 @@
+ISSUES
+======
+JSON schema validation:
+ 
+  - have had to remove the $schema line in order to get the schema validator to work
+          "$schema": "http://json-schema.org/schema#",
+
+
+SANITZE
+========
+
+
+
+
+DX number
+=========
+
+DX number is attached to an address, not a court.  I take the lowest of them alphanetically after excluding emtpy strings and nulls
+
+
```

Comment:
That's correct. I don't expect that many courts have more than one DX address, though.